modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -125,11 +125,9 @@ def apply(self, func: Callable, *args, **kwargs):
         If ``LazyExecution`` is enabled, the function is not applied immediately,
         but is added to the execution tree.
```

Comment:
@AndreyPavlenko should this also be removed?

Reply:
Thanks. To be removed in a separate PR.